app/controllers/greetings_controller.rb

```diff
@@ -1,6 +1,6 @@
 class GreetingsController < ApplicationController
   def random_greeting
-    message = Message.order("RANDOM()").first
+    message = Message.order('RANDOM()').first
     render json: { greeting: message&.content }
   end
```

Comment:
-[ ] In the `seed.rb` file, you're meant to create `5` different greetings which would be used to test your API endpoint to your front end. Kindly so so 👍 

### Requirement:
![image](https://github.com/tonnytech/my_new_rails_app/assets/50990925/1a4be046-f809-45b5-a394-f125ad6f4045)

Reply:
Hi @Shedrack-Sunday,

Thanks for the feedback. However according to the instructions, I'm required to display one random greeting at a time. Check this out:
1. I'm required to create an API endpoint that selects A RANDOM GREETING
![firstGreeting](https://github.com/tonnytech/my_new_rails_app/assets/92226962/4d10b171-0e1f-43a8-be44-8557c393e8e1)

2. I'm required to create a Greeting component that will display a greeting:

![secondG Reeting](https://github.com/tonnytech/my_new_rails_app/assets/92226962/a537e6a0-717a-4735-9a49-7eefabffc875)